service/Core/Handlers/SummarizationHandler.cs

```diff
@@ -132,12 +132,12 @@ private async Task<(string summary, bool skip)> SummarizeAsync(string content, I
     {
         ITextGenerator textGenerator = this._orchestrator.GetTextGenerator();
         int contentLength = textGenerator.CountTokens(content);
-        this._log.LogError("Size of the content to summarize: {0} tokens", contentLength);
+        this._log.LogTrace("Size of the content to summarize: {0} tokens", contentLength);
 
         // If the content is less than 30 tokens don't do anything and move on.
         if (contentLength < MinLength)
         {
-            this._log.LogError("Content is too short to summarize ({0} tokens), nothing to do", contentLength);
+            this._log.LogDebug("Content is too short to summarize ({0} tokens), nothing to do", contentLength);
```

Comment:
this would be better as a warning

Reply:
Would you like to make it warning in the SummarizationParallelHandler as well?

Reply:
yes it would be nice to have them consistent

Reply:
Applied changes in https://github.com/microsoft/kernel-memory/pull/691/commits/7bca3e55f78240d4aae87bf8daa52cf0f2916aa7